views/index.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.html" %}
+
+{% set page_title = "Kainos Jobs" %}
+
+{% block main %}
+ <main>
+        <section>
+            <div class="container-fluid hero">
+              <div class="container-fluid smaller">
+                  <button class="btn mt-2 hero-login" href ="#">Sign in to view jobs</button>
```

Comment:
This button fails in accessibility as it can be hard to read against the background